app/constants/device.js

```diff
@@ -16,5 +17,6 @@ export default {
     ...deviceTypes,
     DOCUMENTS_PATH: `${RNFetchBlobFS.dirs.CacheDir}/Documents`,
     IMAGES_PATH: `${RNFetchBlobFS.dirs.CacheDir}/Images`,
+    IS_IPHONE_X: DeviceInfo.getModel().includes('iPhone X'),
```

Comment:
Strictly speaking, these other constants aren't action types, so we might want to separate them at some point